archetype/engine-v2/src/main/java/io/helidon/build/archetype/engine/v2/ast/Input.java

```diff
@@ -726,6 +919,10 @@ protected Block doBuild() {
                     return new Input.List(this);
                 case OPTION:
                     return new Input.Option(this);
+                case VALIDATIONS:
```

Comment:
Is the intent to have these blocks declared under `<inputs>` ?
Can we create a script that only includes validations and re-use it multiple times ?